examples/utils/bundle-size-tests/src/odspPrefetchSnapshot.ts

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { prefetchLatestSnapshot } from "@fluidframework/odsp-driver";
+
+export function apisToBundle() {
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    prefetchLatestSnapshot(undefined as any, undefined as any, undefined as any, undefined as any, undefined);
```

Comment:
Emails suggested that this produced 30kb bundle. Is this really the case? There is not much code in here.
It's worth taking a look on why it's so large. I assume numbers are for non-minimized code, right?

Reply:
It should be minified code. I would recommend having someone take a look of the output of a plugin like this when we run just this api through webpack: https://github.com/webpack-contrib/webpack-bundle-analyzer

Reply:
This file seems to be about ~10Kb and then almost all utilities are used from odspUtils which also seems to account for ~10KB and then there are some other imports from driver-defn, assert, uuid etc which could have accounted for 10Kb more.

Reply:
@heliocliu These are non-minimized bundle size, right?

Reply:
The metric is for minified code.  The 30kb includes all its dependencies (and their dependencies, etc) as well.  If a host were to take a dependency on this method without already having any dependencies on this method's dependencies, the host would incur a 30kb size increase.  Like mentioned before, the tool doesn't seem to handle new metrics well, but the way it produces the numbers in the table is otherwise simple and the difference in the Total Size Size Diff value and the sum of individual Size Diff values gives the size of the newly added metrics

Reply:
I replied in the email with division of sizes. Just pasting here but we can continue discussion in email thread.

Source | Size(KB)
-- | --
OdspPrefetchSnapshot(Main code file) | 2.72
Common-utils | 3.86
uuid | 1.14
node-fetch | 0.325
ms/index.js | 1.38
node_modules/debug/src | 4.26
Telemetry utils | 5.05
odsp-doclib-utils | 3.5
driver-utils | 4.16
Driver-defn | 2.07
  | 28.465